README.md

```diff
@@ -159,7 +159,7 @@ The Cassandra Mesos Framework is a maven project with modules for the Framework,
 ### Install Maven
 The Cassandra Mesos Framework requires an install of Maven 3.2.x.
 
-### Setup maven toolchain for protoc
+### Setup Maven toolchain for protobuf
```

Comment:
Needs to stay protoc, since the protobuf jar is included via maven, but the proto compiler (protoc) is a binary that has to be installed on the host.